lcls-twincat-pmps/PMPS/GVLs/PMPS_GVL.TcGVL

```diff
@@ -61,7 +62,9 @@ VAR_GLOBAL CONSTANT
         nTran := 1,
         neVRange :=  2#1111_1111_1111_1111_1111_1111_1111_1111,
         nRate    := UDINT#1000000,
-        astApertures := [(Width:=1E3, Height:=1E3), (Width:=1E3, Height:=1E3),(Width:=1E3, Height:=1E3),(Width:=1E3, Height:=1E3)]
+        astApertures := [(Width:=1E3, Height:=1E3), (Width:=1E3, Height:=1E3),(Width:=1E3, Height:=1E3),(Width:=1E3, Height:=1E3)],
+		astAttenuators :=  [PMPS_GVL.AUX_ATTENUATORS(stAttenuators)]
```

Comment:
I've never seen this syntax before. I think it's neat even though I don't understand it at all:
`[UINT(struct)]` ?

Reply:
It's a way to initialize the array 
e.g. arr2 : ARRAY [1..2,3..4] OF INT := 1,3(7); (* short for 1,7,7,7 *)
instead of having to do it for all 16 rows. (PMPS_GVL.AUX_ATTENUATORS)

Reply:
Interesting! I didn't know that was an option!

Reply:
Thanks for the explanation - neat...